README.md

```diff
@@ -1,5 +1,117 @@
-#Building Responsive Sites
+# Building Responsive Sites
 
-<iframe width="640" height="480" src="//www.youtube.com/embed/T649edELGoY?rel=0&modestbranding=1" frameborder="0" allowfullscreen></iframe><p><a href="https://www.youtube.com/watch?v=T649edELGoY">Building Responsive Sites</a></p>
+## Problem Statement
+On the modern web, we frequently access the same content on a variety of
+platforms. Each device will display content differently due to their sizes
+and proportions, causing text to shrink, or parts of the page to shift around,
+and can lead to poor user experience. As front end developers, we want our 
+web sites to look good and function well regardless of whether a user is on 
+their laptop, phone or tablet. This has been an ongoing challenge, so what 
+strategies have been developed to address these issues?
+
+## Objectives
+
+1. Using server-side detection for redirection
+2. Creating responsive layouts with CSS
+3. What is desktop-down design
+3. What is mobile-up design
+4. Explain graceful degradation
+5. Explain progressive enhancement
+6. Why make users update their browsers
+
+## Using Server-Side Detection for Redirection
+
+It is possible for web servers to check the requesting device's headers info,
+and send back files specific to the type of device. Going to
+[twitter.com](twitter.com) on your phone will automatically redirect you to
+[mobile.twitter.com](mobile.twitter.com) because their servers detected that a
+phone was sending the request.
+
+Unfortunately, while this is a common solution, it requires building _two_
+separate websites. Every time you want to change something about the content
+of your site, you have to do it twice, once for both the desktop and mobile
+versions of your site.
+
+## Creating Responsive Layouts with CSS
+
+The other common solution to is to use Cascading Style Sheets.  Instead of a
+server detecting the type of device, every user receives the same content,
+and the provided CSS informs the user's browser on how to style the DOM. A well
+designed, responsive website relies on a combination of CSS media queries,
```

Comment:
1. Probably need to include " this is the approach we want"
2. is this last sentence worth keeping. they don't know most of the words here and we don't go on to explain it later...